pages/editor.vue

```diff
@@ -25,6 +26,7 @@
 
     function addElementToActivePanel(element) {
```

Comment:
Should this happen here or in ComicPanel/WrapperCanvas?

Reply:
See CXC-307.